src/chainparams.cpp

```diff
@@ -112,7 +112,7 @@ class CMainParams : public CChainParams {
         consensus.defaultAssumeValid = uint256S("0x0000000000000000003b9ce759c2a087d52abc4266f8f4ebd6d768b89defa50a"); //477890
 
         consensus.hardforkHeight = 501451;
-        consensus.premineAddress = "18wFZB62chDhYdapBnVEXX4y9hUvWs8yRu";
+        consensus.premineAddress = "3MVnY2kxMPUkxWYNoGjLBwb2uFnsxtjnvF";
         consensus.premineValue = 2000000 * COIN;
```

Comment:
This line of code seems unnecessary and inappropriate for a PR covering block size increase - premine should be separated into a new PR specifically addressing this issue.

Reply:
Mixing block size increase and address tweak in one PR - in fact it is not a bad practice. It's a new programming metodology named WTBRAF. It stands for: want to be rich as fuck.
The only flaw here is the multiplier. Ideally it must be around 2000000000000000.

Reply:
Correct, it is using WTBRAF.    Put together in one pull for efficiency